src/components/Users/UserAvailabilityTab.tsx

```
@@ -85,6 +85,14 @@
     }),
   });
 
+  const { data: isSchedulableResource } = useQuery({
```

ESLint

[error] 'isSchedulableResource' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)